File: Modul 7/Soal 2/PRAK602-2210817220018-Piola Evania.c
#include <stdio.h>

int main()
{
    for(int i = 1; i < 4; i++){
        int b, isi;
        scanf("%d", &b);
        int m[b];
        for(int i = 0; i < b; i++){
            scanf("%d", &isi);
            m[i] = isi;
        }
        printf("\n");
        for(int i = 0; i < b; i++){
            printf("%d ", m[i] * (i + 1));
        }
        printf("\n\n");
    }
}

File: Modul 7/Soal 2/PRAK602-2210817220018-Piola Evania.py
for i in range(3):
    b = int(input())
    isi = list(map(int, input().split()))
    m = []
    a = 0
    for i in range(0, b):
        m.append(isi[a : a + b])
    for i in range(0, b):
        print(m[0][i] * (i + 1), end=' ')
    print("\n\n")